### PR TITLE
Small documentation improvement

### PR DESCRIPTION
**Pull request by piotr-gawron**

### CODE REVIEW

File: development.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Development
+
+Before you start here is a list of required dependencies:
+
+* jdk 8
+* scala 2
```
Comment (sherzinger) [inline development.md]: 2.11.12 specifically. Scalas minor version are not backwards compatible

**Comment by sherzinger** [comment]
Thank you for the docs. Can you update the scala version (s. comment)? I'll merge after.

Generally speaking, I'd like to keep the docs as platform independent as possible. Ada doesn't specifically target Ubuntu, so apt commands in the docs should be in the debian/ubuntu sub section, or maybe just mentioned as a version.

**Comment by piotr-gawron** [comment]
> Generally speaking, I'd like to keep the docs as platform independent as possible. Ada doesn't specifically target Ubuntu, so apt commands in the docs should be in the debian/ubuntu sub section, or maybe just mentioned as a version.


apt-get command are in section
>## Linux (Ubuntu) guidelines

I was hoping it's enough

**Comment by sherzinger** [comment]
My apologies. I skipped that part. You're absolutely right

**Pull request merged**